testnets/holesky/presets/sidecar-delegations-preset.env.example

```diff
@@ -0,0 +1,60 @@
+# Ethereum Node Connections + PBS URLs
+
+# Port to listen on for incoming JSON-RPC requests of the Commitments API. This
+# port should be open on your firewall in order to receive external requests!
+BOLT_SIDECAR_PORT=8017
+# Execution client API URL
+BOLT_SIDECAR_EXECUTION_API_URL="http://172.56.0.1:8545"
+# URL for the beacon client
+BOLT_SIDECAR_BEACON_API_URL="http://172.56.0.1:5052"
+# Execution client Engine API URL. This is needed for fallback block building
+# and must be a synced Geth node
+BOLT_SIDECAR_ENGINE_API_URL="http://172.56.0.1:8551"
+# The port from which the Bolt sidecar will receive Builder-API requests from the Beacon client
+BOLT_SIDECAR_CONSTRAINTS_PROXY_PORT=18550
+# URL to forward the constraints produced by the Bolt sidecar to a server
+# supporting the Constraints API, such as an MEV-Boost fork
+BOLT_SIDECAR_CONSTRAINTS_API_URL="http://bolt-mev-boost-holesky:18551"
+# The JWT secret token to authenticate calls to the engine API. It can be
+# either be a hex-encoded string or a file path to a file containing the
+# hex-encoded secret.
+BOLT_SIDECAR_ENGINE_JWT_HEX=
+# The fee recipient address for fallback blocks
+BOLT_SIDECAR_FEE_RECIPIENT=
+# Secret ECDSA key used to sign commitment messages on behalf of your validators. The public
+# key associated is an operator registered in the `BoltManager` contract and in a restaking
+# protocol.
+BOLT_SIDECAR_COMMITMENT_PRIVATE_KEY=
+# Secret BLS key to sign fallback payloads with. This can be any key. You can generate one
+# with the `bolt` CLI tool, using `bolt generate bls`.
+BOLT_SIDECAR_BUILDER_PRIVATE_KEY=
```

Comment:
I am thinking that we should use a random one as default and not expose this variable in the preset, because in 99% of cases validators don't care about this and if they do it's OK for it to be an advanced feature IMO.

If you agree I can draft an issue

Reply:
I agree with the point you make. Why would you need to draft an issue for it?

Reply:
because BOLT_SIDECAR_BUILDER_PRIVATE_KEY currently has no default value in the sidecar, we need to add it

Reply:
it can be done here as we already have a Default impl, gtg